### PR TITLE
Repleace request library with node-fetch

### LINKED ISSUE #12: library request is deprecated
please replace the library request which is deprecated with node-fetch for example.

### PR DESCRIPTION
**Pull request by codac**
As the request library is deprecated, it needs to get replaced by node-fetch!
Please help me testing it.

### CODE REVIEW

**Comment by master117** [comment]
Looks good. Bracket style changed but since I'm not maintaining it too actively that doesn't matter.

**Comment by master117** [comment]
While it doesn't matter, you should use the .catch of promises instead of try catches inside then then.

File: node_helper.js [hunk 1]
```diff
@@ -20,92 +18,77 @@ module.exports = NodeHelper.create(
...
-                        }
-                    };
+                    }
 
-                    request(options,
-                        function (error, response, body)
-                        {
+                    fetch('http://api.football-data.org/v2/competitions/' + league.toString(), {
+                          options
+                    }).then(function (error, response, body) {
```
Comment (master117) [inline node_helper.js:33]: While it doesn't matter, you should use the .catch of promises instead of try catches inside the then.

Comment (codac) [reply node_helper.js:33]: true

**Pull request merged**